armi/utils/pathTools.py

```diff
@@ -240,7 +240,7 @@ def cleanPath(path, mpiRank=0):
     """
     valid = False
     if not os.path.exists(path):
-        if context.MPI_SIZE > 1:
+        if context.MPI_SIZE > 1 and context.MPI_DISTRIBUTABLE and barrier:
```

Comment:
For computational efficiency, can we write this as:

> if barrier and context.MPI_SIZE > 1 and context.MPI_DISTRIBUTABLE:

The idea here is there is no real cost to checking the value of `barrier`, but there is an incurred cost to pulling up the `context` object and checking the namespace inside it.

This is not super important, but it's also easy to fix.